src/components/NoteComponent.vue

```diff
@@ -43,7 +43,7 @@ export default {
 }
 </script>
 
-<style>
+<style scoped>
```

Comment:
Added scoped to a lot of components here. I think while we are building out the initial site for release, we should use `scoped` on our component's styles. The reason is that the more we build out without scoping it, the harder it will be to find where the styles are coming from and preventing our customs.

In the future when we have time, we can always go back and abstract out duplication to tackle the tech debt

Reply:
yeah, not sure why these weren't scoped - probably because Doug built them. I always use scoped